modules/Bitfinex.py

```diff
@@ -30,6 +30,9 @@ def __init__(self, cfg, log):
         self.symbols = []
         self.ticker = {}
         self.tickerTime = 0
+	self.baseCurrencies = ['USD', 'BTC', 'ETH']
```

Comment:
I like what you did in the Bitfinex module, making it easy to add new base currencies is a useful change.